org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/Filters.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2017 Mountainminds GmbH & Co. KG and Contributors
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Evgeny Mandrikov - initial API and implementation
+ *
+ *******************************************************************************/
+package org.jacoco.core.internal.analysis.filter;
+
+import org.objectweb.asm.tree.MethodNode;
+
+/**
+ * Filter that combines all other filters.
+ */
+public final class Filters implements IFilter {
+
+	public static final IFilter ALL = new Filters();
```

Comment:
This introduces the only warning in core: public field needs JavaDoc

Reply:
Done.